src/Command/ReconcileCommand.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+
+namespace Webgriffe\SyliusAkeneoPlugin\Command;
+
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Command\LockableTrait;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Webgriffe\SyliusAkeneoPlugin\ImporterInterface;
+use Webgriffe\SyliusAkeneoPlugin\ReconcilerRegistryInterface;
+use Webgriffe\SyliusAkeneoPlugin\ReconcilerInterface;
+use Webmozart\Assert\Assert;
+
+/**
+ * @psalm-suppress PropertyNotSetInConstructor $lock
+ */
+final class ReconcileCommand extends Command
+{
+    use LockableTrait;
+
+    public const SUCCESS = 0;
+
+    public const FAILURE = 1;
+
+    private const IMPORTER_OPTION_NAME = 'importer';
+
+    protected static $defaultName = 'webgriffe:akeneo:reconcile';
+
+    /**
+     * @var ReconcilerRegistryInterface
+     */
+    private $reconciliationRegistry;
+
+    /**
+     * ReconcileCommand constructor.
+     * @param ReconcilerRegistryInterface $reconciliationRegistry
+     */
+    public function __construct(ReconcilerRegistryInterface $reconciliationRegistry)
+    {
+        parent::__construct();
+        $this->reconciliationRegistry = $reconciliationRegistry;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setDescription('Replicates the status of Akeneo products on Sylius.')
+            ->setHelp('This command allows you to reconcile the Akeneo\'s products status with the current on Sylius.')
```

Comment:
A reconciler is generic. It could also be used for something else than products. I'd also remove the reference to the "status" because this is an implementation details of the product reconciler
So I'd provide a generic description and I'd remove the help:
```suggestion
            ->setDescription('Reconciles Akeneo entities on Sylius.')
```